openapi/components/schemas/GatewayAccountConfig/Orbital.yaml

```
@@ -27,3 +27,10 @@ allOf:
               - USD
               - EUR
               - GBP
+          mainCurrency:
```

Won't it be confusing with targetCurrency?

```suggestion
          mainCryptoCurrency:
```